### PR TITLE
fix: auth trigger params parsing and test wait lines

### PR DESCRIPTION
**Pull request by edwardfoyle**
#### Description of changes

Fixes a wait line in a test and fixes the updatingAuth object that hangs off of context to read from build/parameters.json instead of cli-inputs.json

#### Description of how you validated changes
Manually tested and ran the e2e test locally. The test failed due to not having some env vars set up right, but should be g2g.

#### Checklist

- [ ] PR description included
- [ ] `yarn test` passes
- [ ] Tests are [changed or added](https://github.com/aws-amplify/amplify-cli/blob/master/CONTRIBUTING.md#tests)
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] New AWS SDK calls or CloudFormation actions have been added to relevant test and service IAM policies

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/amplify-category-auth/src/commands/auth/update.ts [hunk 1]
```diff
@@ -53,8 +53,8 @@ export const run = async (context: $TSContext) => {
   }
   const resourceName = await getAuthResourceName(context);
   await checkAuthResourceMigration(context, resourceName);
-  const cliState = new AuthInputState(resourceName);
-  context.updatingAuth = await cliState.loadResourceParameters(context, cliState.getCLIInputPayload());
+  const providerPlugin = context.amplify.getPluginInstance(context, servicesMetadata.Cognito.provider);
```
Comment (akshbhu) [inline packages/amplify-category-auth/src/commands/auth/update.ts:56]: we also need the JSON.parse(providerPlugin.loadResourceParameters(context, 'auth', resourceName))

Comment (edwardfoyle) [reply packages/amplify-category-auth/src/commands/auth/update.ts:56]: No, I ran it with this and it works. The line that's already there is
```ts
const previouslySaved =
    typeof context.updatingAuth.triggers === 'string' ? JSON.parse(context.updatingAuth.triggers) : context.updatingAuth.triggers;
```
And with this change `context.updateAuth.triggers` will be a string and be properly parsed

Comment (akshbhu) [reply packages/amplify-category-auth/src/commands/auth/update.ts:56]: okay

**Review by akshbhu — APPROVED** [review]

File: packages/amplify-category-auth/src/commands/auth/update.ts (hunk 1 above)
Comment (edwardfoyle) [reply packages/amplify-category-auth/src/commands/auth/update.ts:56]: oh wait, I may not have rebuilt in between. Let me check again...

Comment (edwardfoyle) [reply packages/amplify-category-auth/src/commands/auth/update.ts:56]: yep good catch, I hadn't rebuilt and it is needed. Updating the PR now

**Comment by ammarkarachi** [comment]
catthumbs

**Review by ammarkarachi — APPROVED** [review]

**Pull request merged**